client/src/app/site/services/global-search/global-search.service.ts

```diff
@@ -190,7 +190,7 @@ export class GlobalSearchService {
         };
     }
 
-    private getText(collection: string, result: GlobalSearchResponseEntry, results: GlobalSearchResponse) {
+    private getText(collection: string, result: GlobalSearchResponseEntry, results: GlobalSearchResponse): any {
```

Comment:
I don't like that the `getText` function returns any, would it be possible to make it more precise to find the possible values of `GlobalSearchResponseEntry.content` and maybe cast the data/throw an error if it isn't as expected and maybe edit the `GlobalSearchResponseEntry` interface to be more precise?